### ISSUE TITLE
Privacy issue: googleapis font http requests

### ISSUE THREAD

**Issue by danowar2k** [h5p/h5p-editor-php-library#80]
Would it be possible to store the googleapis fonts css and the fonts themselves locally in the repository instead of requesting them? 

It has come to my attention that the requests to the google infrastructure lets google gather information about where and how much their fonts are used in h5p.

At the moment I have to hack the Moodle plugin to use local resources and I would rather not have hacks in place.

**Comment by icc** [comment]
Yes, we do aim for all fonts to be locally stored within the plugin or content type. Have you come across someplace where this is not the case?

**Comment by danowar2k** [comment]
Well, in my case here:
https://github.com/h5p/h5p-editor-php-library/blob/master/styles/css/h5p-hub-client.css

But when I search for "googleapis", I found more:
https://github.com/search?q=org%3Ah5p+googleapis&type=Code

**Comment by icc** [comment]
Thank you, I've created an issue in our own tracker for cleaning up this: [HFP-2759](https://h5ptechnology.atlassian.net/browse/HFP-2759)

**Comment by danowar2k** [comment]
Quick question: I have added a local hack that downloads local versions of these fonts for now.

I'm trying to test whether my local fonts are used. I removed all local() alternatives in the CSS.

Are the fonts loaded when they are used or always? I can't seem to find them in my Chrome developer tools.

It also seems that each defined font is only used for certain character ranges.  An example:

```
@font-face {
  font-family: 'Open Sans';
  font-style: normal;
  font-weight: 400;
  src: url(./fonts/cyrillic-ext.woff2) format('woff2');
  unicode-range: U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;
}
```

How can I test this? Are those special fonts only used when I change the language to Russian (cyrillic) etc.?

There are also fonts for latin, though...

**Comment by danowar2k** [comment]
Is there anything I can do to help? Who gets to call how the change will be implemented? Where should the fonts be stored etc.?

**Comment by jankapunkt** [comment]
Any update on this?

**Comment by danowar2k** [comment]
I think this was/is fixed by the commits for #139
